tools/RAiDER/cli/raiderDelay.py

```diff
@@ -244,7 +242,11 @@ def read_template_file(fname):
         if key == 'date_group':
             template['date_list'] = parse_dates(AttributeDict(value))
         if key == 'aoi_group':
-            template['aoi'] = get_query_region(AttributeDict(value))
+            ## in case a DEM is passed and should be used
```

Comment:
@bbuzz31 I'm wondering if we should flatten the parameter dict once it gets read. That way we don't have to worry about groups. It's complicated since some things depend on other things. Is that what you're doing here?

Reply:
that's not a bad idea. I guess would just have to make sure there aren't duplicate parameters.

And yeah this case is an ugly hack around for the use case when someone specifies a bounding box and use dem lat lon. 

Another option would be to move the use_dem param to the AOI group (possibly should do this anyway)